planner/core/src/main/java/eu/seaclouds/platform/planner/core/DamGenerator.java

```diff
@@ -91,6 +102,7 @@ public static String generateDam(String adp, String monitorGenURL, String monito
         adpYaml = DamGenerator.addApplicationInfo(adpYaml, slaInfoResponse, SLA_INFO_GROUPNAME);
 
         Map groups = (Map) adpYaml.remove(GROUPS);
+        adpYaml.remove(NODE_TYPES);
```

Comment:
@andreaturli yes I'm aware of #252 and of the many changes that is carries at this point. I think the MonitorDamGenerator is not affected by #252 since this PR just changes the way information produced by the monitorDamGenerator are used, but the information itself is unchanged. What I'm missing is how the removal of NodeTypes from the current ADP (actually, at least so far, the node types have always been there) is connected with #252. Can you clarify me this point? I'm asking because on the other hand the removal of node types from the ADP would instead affect the MonitorDamGenerator.